install.sh

```diff
@@ -41,14 +72,14 @@ url="https://fcli-binaries.s3.eu-west-1.amazonaws.com/channels/stable/${archive}
 
 echo "Downloading and extracting Fluence archive..."
 mkdir -p "${FLUENCE_USER_DIR}/cli"
-wget -q --show-progress -O "${TEMP}/${archive}" "$url"
+curl --progress-bar -o "${TEMP}/${archive}" "$url"
 tar --strip-components=1 -xzf "${TEMP}/${archive}" -C "${FLUENCE_USER_DIR}/cli"
 
 echo "Fluence CLI installation complete!"
-if echo $PATH | grep -q $HOME/.local/bin ; then
+if echo $PATH | grep -q $HOME/.local/bin; then
   mkdir -p $HOME/.local/bin
   ln -sf ${FLUENCE_USER_DIR}/cli/bin/fluence $HOME/.local/bin/fluence
 else
-  echo "Create a symlink to fluence binary in any directory that is in \$PATH, for example:"
-  echo "sudo ln -s ${FLUENCE_USER_DIR}/cli/bin/fluence /usr/local/bin/fluence"
+  echo "Adding fluence cli to \$PATH with 'sudo ln -s ${FLUENCE_USER_DIR}/cli/bin/fluence /usr/local/bin/fluence"
+  sudo ln -s ${FLUENCE_USER_DIR}/cli/bin/fluence /usr/local/bin/fluence
```

Comment:
![image](https://github.com/fluencelabs/cli/assets/22859527/313d9436-5d31-4be3-9863-9f1166f8b95b)
Maybe call to `ln -sf` here also, so as not to exit with the error?